runtime/caml/ui.h

```diff
@@ -18,15 +18,11 @@
 #ifndef CAML_UI_H
 #define CAML_UI_H
 
-#ifdef CAML_INTERNALS
-
```

Comment:
Is the removal of this guard correct?

Reply:
The whole `runtime/caml/ui.h` should be removed, as well as its inclusion in `runtime/startup_nat.h`.  It's a leftover from very old times.

Reply:
Tracking this for post-merge fixing in https://github.com/ocaml/ocaml/issues/10861

Reply:
Tracking this for post-merge fixing in https://github.com/ocaml/ocaml/issues/10861